scripts/build.py

```diff
@@ -806,6 +824,9 @@ def get_file_hashes(directory_path):
             filepath = os.path.join(root, filename)
             if should_file_be_built(filepath) and not any(
                     filename.endswith(p) for p in FILE_EXTENSIONS_TO_IGNORE):
+                # In case the path in in Windows-style, We need to convert the
```

Comment:
Same thing -- I suspect the explanation would be similar to https://github.com/oppia/oppia/pull/8139/commits/b90ee3045f3fc810a61d42123808587f29a87eb3#diff-191be61dc824545e1299bcaae58a8b58R731 if both relate to hashes.json? 

Ditto elsewhere. Use the same comment if the justification is the same. Perhaps a cleaner way to do this is to add a long explanatory docstring above HASHES_JSON_FILENAME line explaining clearly what the format of the file is and why, and then every other comment can be "The filepaths in hashes.json need to be posixpath style. See the comment above HASHES_JSON_FILENAME for details."

Reply:
Updated